api/room/schema.py

```diff
@@ -88,7 +88,14 @@ def mutate(self, info, office_id, **kwargs):
 
         admin_roles.create_rooms_update_delete_office(office_id)
         query = Room.get_query(info)
-        exact_query = room_join_office(query)
+        active_rooms = query.filter(RoomModel.state == "active")
+        query_result = active_rooms.filter(
+            RoomModel.calendar_id == kwargs.get('calendar_id'))
+        if query_result.count() > 0:
```

Comment:
Hello@dnuwa, good work. I have a suggestion, After that suggested code by @michael-basweti, how about we write the check as `if query_result.count():` Instead of `if query_result.count() > 0:`. It works the same.

Reply:
@nakatuddesuzan this change has been implemented